src_ext/patches/cmdliner.common/0001-Restore-4.02.3-support.patch

```diff
@@ -21,10 +21,11 @@ diff -Naur a/pkg/META b/pkg/META
 diff -Naur a/src/cmdliner.ml b/src/cmdliner.ml
 --- a/src/cmdliner.ml	2019-06-14 09:55:03.000000000 +0000
 +++ b/src/cmdliner.ml	2019-11-21 14:29:33.236981936 +0000
-@@ -4,11 +4,11 @@
+@@ -4,11 +4,12 @@
```

Comment:
The changes in this patch are actually a consequence of Result 1.5 which (slightly annoyingly) sync'd the entire Result module for OCaml 4.08+ rather than just the type (annoying in the sense that I'd have made it version 2.0...)